CHANGES.rst

```diff
@@ -1,11 +1,30 @@
 Changes
 =======
 
-Version master (UNRELEASED)
+Version 0.7.0 (2020-10-19)
 ---------------------------
 
-- Adds local users forms for sign in and sign up functionalites.
-
+- Adds user profile page.
+- Adds local user forms for sign-in and sign-up functionalities.
+- Adds home page suitable for standalone vs CERN deployments.
+- Adds page refresh button to workflow detailed page.
+- Adds favicon to the web interface pages.
+- Adds basic theme scaffolding.
+- Adds announcement configuration to easily display messages on the web interface.
+- Adds pagination on the workflow list page.
+- Fixes loading workflow indicator.
+- Fixes displaying of non-existing workflows.
+- Fixes file preview functionality experience to allow/disallow certain file formats.
+- Fixes workflow specification display to show runtime parameters.
+- Fixes display of footer links in case they are not set during deployment.
+- Changes configuration to dynamically detect URL.
+- Changes main loader of the web interface.
+- Changes workflow list page to use hooks.
```

Comment:
In the end we're using hooks everywhere, our codebase is 100% class-free, only functions. So this line could more general.